webapp/src/ts/services/language.service.ts

```diff
@@ -0,0 +1,93 @@
+import * as moment from 'moment';
+import { Injectable } from '@angular/core';
+import { CookieService } from 'ngx-cookie-service';
+import { TranslateService } from '@ngx-translate/core';
+import { SettingsService } from './settings.service';
+import { UserSettingsService } from './user-settings.service';
+
+const localeCookieKey = 'locale';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SetLanguageCookieService {
+  constructor(
+    private cookieService:CookieService
+  ) {
+  }
+
+  set(value) {
+    this.cookieService.set(localeCookieKey, value, 365, '/');
+    return value;
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SetLanguageService {
```

Comment:
These two services are related, but what do you think about having them in separate files? So, we find SetLanguageService in set-language.service.ts to keep consistency with other services and also to unit test them separately.

Reply:
I think the whole idea was to have them in the same file so we don't have to declare the `localeCookieKey` somewhere else. 
I think other people (Mariano and Njuguna) have migrated this file and I feel like we should just leave it like this in this iteration.